lib/bolt/task.rb

```diff
@@ -82,7 +82,7 @@ def implementations
     # and any additional files (name and path)
     def select_implementation(target, provided_features = [])
       impl = if (impls = implementations)
-               available_features = target.feature_set + provided_features
+               available_features = target.features.to_set + provided_features
```

Comment:
Why was this change necessary?

Reply:
It wasn't. This was early on in my spec-fixing adventure, before I realized stubs needed to be updated. This is the part that I needed to fix: https://github.com/puppetlabs/bolt/blob/master/spec/bolt/task_spec.rb#L22

It's in the update.